src/DDLQueryBuilder.php

```diff
@@ -314,4 +314,15 @@ private function dropConstraintsForColumn(string $table, string $column, string
     EXEC (N'ALTER TABLE ' + @tableName + ' DROP CONSTRAINT [' + @constraintName + ']')
 END";
     }
+
+    /**
+     * @throws NotSupportedException MSSQL doesn't support cascade drop table.
+     */
+    public function dropTable(string $table, bool $ifExists = false, bool $cascade = false): string
+    {
+        if ($cascade) {
+            throw new NotSupportedException('MSSQL doesn\'t support cascade drop table.');
```

Comment:
It is OK for now, but maybe we should traverse schema and do delete ourselves in this case for portability reasons.